### PR TITLE
Progress in F1 non-BLT artifacts

### PR DESCRIPTION
**Pull request by rusefillc**
#212 and more better artifacts for github action

### CODE REVIEW

**Comment by mck1117** [comment]
image

**Review by mck1117 — CHANGES_REQUESTED** [review]
image

File: .github/workflows/build-firmware.yaml [hunk 1]
```diff
@@ -42,4 +46,4 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: Wideband ${{matrix.build-target}}
-          path: ./firmware/build/wideband.*
+          path: ./firmware/c/${{matrix.build-target}}/wideband.*
```
Comment (rusefillc) [inline .github/workflows/build-firmware.yaml:49]: the typo is here

**Comment by rusefillc** [comment]
typo in f1 script fixed, f0 script is now a bit more consistent with f1

**Comment by mck1117** [comment]
If this overlaps with now-merged #214, I'm happy to modify the F0 script to place output files as I theoretically maintain that board..

**Comment by rusefillc** [comment]
@mck1117 you mean https://github.com/mck1117/wideband/pull/217?

**Comment by mck1117** [comment]
oops, 43def3f98257850f7820503f0984cda0e3c078d0 implemented f0 in master

**Comment by rusefillc** [comment]
@mck1117 only F1 progress now here

**Pull request merged**